dockstore-common/src/main/scala/io/dockstore/common/WdlBridge.scala

```diff
@@ -371,3 +369,11 @@ case class MapResolver() extends ImportResolver {
 
   override def cleanupIfNecessary(): ErrorOr[Unit] = ().validNel
 }
+
+object WdlBridgeShutDown {
+  private val logger = LoggerFactory.getLogger("WdlBridge")
+  def shutdownSTTP(): Unit = {
```

Comment:
was that meant to be shutdownHTTP?

Reply:
STTP is what's the library that's being used https://github.com/softwaremill/sttp

I can't do HttpResolver.closeBackendIfNecessary() after resolving imports because it appears it opens once and only once.  So if the client calls it again, it's gone and can't be re-opened.

Reply:
Interesting, that would be a problem if we're re-using the code for resolving imports in the webservice.

Reply:
Cromwell appears to shutdown it along with everything else at the end:

https://github.com/broadinstitute/cromwell/blob/45/engine/src/main/scala/cromwell/server/CromwellShutdown.scala

Reply:
Ah, good find